tests/utils.go

```diff
@@ -1088,6 +1144,87 @@ func AddDownwardMetricsVolume(vmi *v1.VirtualMachineInstance, volumeName string)
 	})
 }
 
+func GetDownwardMetrics(vmi *v1.VirtualMachineInstance) (*api.Metrics, error) {
```

Comment:
The following funcs look like duplicates of [the libinfra funcs](https://github.com/kubevirt/kubevirt/blob/2e6f4848622acce160adb2f8753444a84baccb54/tests/libinfra/metrics.go#L39)

Reply:
They are leftover and not used functions. Thank you, will drop them.